automation/repeated_test.sh

```diff
@@ -157,7 +157,7 @@ if [[ -z ${NEW_TESTS-} ]]; then
     set -e
 
     # skip certain tests for now, as we don't have a strategy currently
-    NEW_TESTS=$(echo "$NEW_TESTS" | sed -E 's/\|?[^\|]*(sriov|multus|windows|gpu|mdev)[^\|]*//g' | sed 's/^|//')
+    NEW_TESTS=$(echo "$NEW_TESTS" | sed -E 's/\|?[^\|]*(sriov|multus|windows|gpu|mdev|sig-sdn)[^\|]*//g' | sed 's/^|//')
```

Comment:
Test files touched: tests/decorators/decorators.go,tests/flags/flags.go,tests/framework/matcher/getter.go,tests/framework/matcher/status.go,tests/sdn/framework.go,tests/sdn/persistentips.go,tests/testsuite/kubevirtresource.go

it wont work with sig-snd iiuc, because it need to skip based on file path
`sdn` might be fine, unless it is too generic and can omit in the future some real files
so you can also consider `persistentips` please